web/ui/react-app/src/pages/targets/EndpointLink.tsx

```diff
@@ -8,23 +8,22 @@ export interface EndpointLinkProps {
 
 const EndpointLink: FC<EndpointLinkProps> = ({ endpoint, globalUrl }) => {
   let url: URL;
+  let search = '';
+  let invalidURL = false;
   try {
     url = new URL(endpoint);
   } catch (err: unknown) {
-    const error = err as Error;
-    return (
-      <Alert color="danger">
-        <strong>Error:</strong> {error.message}
-      </Alert>
-    );
+    invalidURL = true;
+    if (endpoint.indexOf('?') > -1) search = endpoint.substring(endpoint.indexOf('?'));
```

Comment:
Style-wise for increased clarity, let's stick to always having curly braces for conditional blocks and also spreading it over multiple lines, like:

```suggestion
    if (endpoint.indexOf('?') > -1) {
      search = endpoint.substring(endpoint.indexOf('?'));
    }
```

Reply:
Fixed in last commit.